### ISSUE TITLE
[ Bug ] Album still showing in Downloads after deleting all songs

### ISSUE THREAD

**Issue by SindreKjelsrud** [kepelet/flo#32]
I wanted to remove an album from my offline downloads, and couldn't find an easy way to do it except deleting every downloaded song in the album. But the album is still showing, even though it's empty. :sweat_smile: 

After optimizing local storage, by deleting **_all_** downloaded albums and songs, it removed it.

**Comment by faultables** [comment]
yeah this is somewhat annoying. this should be a feature ("remove downloads" in album) to fix this bug lol gonna make it in 1.5

**Comment by faultables** [comment]
Resolved in 1.6
